test/core/expectations/test_expect_not_to_be_null.pkb

```diff
@@ -195,5 +255,40 @@ is
         ut.expect(anydata.convertCollection(ut3.ut_expectation_processor.get_failed_expectations())).not_to_be_empty();
     end;
 
+    procedure null_anydata
+    is
+    begin
+        --Act
+        execute immediate expectations_helpers.unary_expectation_block('not_to_be_null', 'anydata', 'null');
+        --Assert
+        ut.expect(anydata.convertCollection(ut3.ut_expectation_processor.get_failed_expectations())).not_to_be_empty();
```

Comment:
I think we should change that behavior and throw an exception when NULL anydata is passed.